dataminer/Administrator_guide/Databases/Indexing_database/OpenSearch_Database/Installing_OpenSearch_database.md

```diff
@@ -474,3 +477,35 @@ To monitor expiry dates, we recommend using the **Skyline SSL Certificate Monito
 1. [Create an alarm template](xref:Creating_an_alarm_template) to monitor the parameter *Remaining Days* of the protocol, and [assign it to the element](xref:Assigning_an_alarm_template).
 
    ![Alarm template configuration](~/dataminer/images/Skyline_SSL_Certificate_Monitor_thresholds.png)
+
+## Renewing the TLS certificates for OpenSearch
+
+To renew your self-signed TLS certificates for an existing OpenSearch database, you will have to replace the existing certificates with newly signed ones. The expired certificates can no longer be used. To generate new certificates, we recommend using the *Generate-TLS-Certificates* script.
+
+1. Generate the certificates using the [Generate-TLS-Certificates](https://github.com/SkylineCommunications/generate-tls-certificates) script on GitHub.
+
+   This script will generate a .p12 file for every node and a .pem admin cert and key file that is used for security configuration settings. The GitHub script will provide you with the keystore and truststore password.
+
+   A rootCA.crt can also be generated by the GitHub script; however, we recommend using the original *rootCA.crt* (the same one used on the previous renewal/certification). If this is not possible, the new *rootCA.crt* will have to be applied on all DataMiner and OpenSearch nodes.
+
+1. Remove the outdated certificates from the OpenSearch nodes.
+
+   These are usually located in the `\etc\opensearch\cert` folder, but not always. You can find the exact location of the certificates in the TLS segment of the *opensearch.yml* file.
+
+1. Place the correct .p12 file in the `\etc\opensearch\cert\` folder on each corresponding node (making sure the .p12 name matches the node name).
+
+1. Update the opensearch.yml file so that *keystore_filepath* and *truststore_filepath* refer to the location of the .p12 files. (For an example, see [Set up TLS in the opensearch.yml file](#set-up-tls-in-the-opensearchyml-file).)
+
+1. Add the password to the *keystore* and *truststore* password fields in *opensearch.yml*.
+
+1. Navigate to the location where the generated *rootCA.crt* file is stored, and copy it.
+
+   On Ubuntu, you can do so using the following command: `sudo cp rootCA.crt /usr/local/share/ca-certificates/`
+
+1. On every Linux OpenSearch node, update the trusted root certificates.
+
+   On Ubuntu, you can do so using the following command: `sudo update-ca-certificates`
+
+1. Verify the *rootCA.crt*. It should return the status OK.
```

Comment:
What if it doesn't return the status OK? Are there any tips we can include as to how a user could solve this?